common-npm-packages/azure-arm-rest-v2/azure-arm-endpoint.ts

```diff
@@ -96,7 +95,7 @@ export class AzureRMEndpoint {
                 }
                 else {
                     this.endpoint.environmentAuthorityUrl = (!!this.endpoint.environmentAuthorityUrl) ? this.endpoint.environmentAuthorityUrl : "https://login.windows.net/";
-                    this.endpoint.activeDirectoryResourceID = this.endpoint.url;
+                    if (!useGraphActiveDirectoryResource) this.endpoint.activeDirectoryResourceID = this.endpoint.url;
```

Comment:
please use braces to keep it shorter lines and more readable.

Reply:
added braces with next commit